kernel/boot/kernel-amd64-knc/boot/start.S

```diff
@@ -112,15 +112,15 @@ _start_bsp:
         // activate the initial page table
         mov     $0xc0000080, %rcx       // select the EFER MSR
         rdmsr                           // read EFER
-        or      $0x101, %rax            // set LME (bit8) and enable SYSCALL (bit0), XXX enable noexecute (bit11)
+        or      $0x901, %rax            // IA-32e mode enable (bit8), SYSCALL (bit0), noexecute (bit11)
```

Comment:
As the EFER and cr0 bits are set on multiple locations, maybe we should have a define for them. This also gives us more room to document the flags.